maven-plugin/src/main/java/org/openmrs/maven/plugins/utility/ModuleInstaller.java

```diff
@@ -50,27 +50,28 @@ public ModuleInstaller(MavenProject mavenProject,
         this.versionsHelper = versionsHelper;
     }
 
-    public void installCoreModules(Server server, boolean isCreatePlatform, DistroProperties properties, DistroHelper distroHelper) throws MojoExecutionException, MojoFailureException {
+    public void installDefaultModules(Server server) throws MojoExecutionException {
```

Comment:
This function is just the "else" section of the old `installCoreModules` function.